core/ts/MaciState.ts

```diff
@@ -1430,30 +181,5 @@ class MaciState {
   }
 }
 
-const genProcessVkSig = (
-  _stateTreeDepth: number,
-  _messageTreeDepth: number,
-  _voteOptionTreeDepth: number,
-  _batchSize: number,
-): bigint => {
-  return (
-    (BigInt(_batchSize) << BigInt(192)) +
-    (BigInt(_stateTreeDepth) << BigInt(128)) +
-    (BigInt(_messageTreeDepth) << BigInt(64)) +
-    BigInt(_voteOptionTreeDepth)
-  );
-};
-
-const genTallyVkSig = (_stateTreeDepth: number, _intStateTreeDepth: number, _voteOptionTreeDepth: number): bigint => {
-  return (
-    (BigInt(_stateTreeDepth) << BigInt(128)) + (BigInt(_intStateTreeDepth) << BigInt(64)) + BigInt(_voteOptionTreeDepth)
-  );
-};
-
-const genSubsidyVkSig = (_stateTreeDepth: number, _intStateTreeDepth: number, _voteOptionTreeDepth: number): bigint => {
-  return (
-    (BigInt(_stateTreeDepth) << BigInt(128)) + (BigInt(_intStateTreeDepth) << BigInt(64)) + BigInt(_voteOptionTreeDepth)
-  );
-};
-
-export { MaxValues, TreeDepths, MaciState, Poll, genProcessVkSig, genTallyVkSig, genSubsidyVkSig };
+// todo: remove re-export `IncrementalQuinTree` it's originally exported by `maci-crypto`
+export { IncrementalQuinTree, MaciState };
```

Comment:
I'm not sure I understand why the export of IncrementalQuinTree was added? also given you added a comment saying that it should be removed

Reply:
somehow after this refactoring MaciState.d.ts file doesn't export IncrementalQuinTree automatically so had to explicitly re-exported